genie-web/src/test/groovy/com/netflix/genie/web/services/impl/JobKillServiceImplSpec.groovy

```diff
@@ -0,0 +1,75 @@
+/*
+ *
+ *  Copyright 2017 Netflix, Inc.
+ *
+ *     Licensed under the Apache License, Version 2.0 (the "License");
+ *     you may not use this file except in compliance with the License.
+ *     You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *     Unless required by applicable law or agreed to in writing, software
+ *     distributed under the License is distributed on an "AS IS" BASIS,
+ *     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *     See the License for the specific language governing permissions and
+ *     limitations under the License.
+ *
+ */
+package com.netflix.genie.web.services.impl
+
+import com.netflix.genie.common.dto.JobStatus
+import com.netflix.genie.test.categories.UnitTest
+import com.netflix.genie.web.services.JobKillServiceV4
+import com.netflix.genie.web.services.JobPersistenceService
+import com.netflix.genie.web.services.JobSearchService
+import org.junit.experimental.categories.Category
+import spock.lang.Specification
+
+/**
+ * Tests for JobKillServiceImpl
+ *
+ * @author standon
+ * @since 4.0.0
+ */
+@Category(UnitTest.class)
+class JobKillServiceImplSpec extends Specification {
+    JobKillServiceV3 jobKillServiceV3
+    JobKillServiceImpl service
+    JobPersistenceService jobPersistenceService
+    JobKillServiceV4 jobKillServiceV4
+    String jobId
+
+    void setup() {
+        jobPersistenceService = Mock()
+        jobKillServiceV4 = Mock()
+        jobKillServiceV3 = Mock()
+        jobId = UUID.randomUUID().toString()
+        service = new JobKillServiceImpl (
+                jobKillServiceV3, \
```

Comment:
What's up with those slashes?

Reply:
Groovy multiple lines format

Reply:
Groovy is different. You cannot break a sting across multiple lines without `\`. I encountered that problem once and adding `\` worked. So this was my logical extension. As explained here - https://stackoverflow.com/questions/5079797/whats-wrong-with-groovy-multi-line-string - `\` is not needed for a constructor as groovy has a context and hence can find that multiple lines belong to the constructor. Removed `\`.

Reply:
This is not a string. Groovy and Java behave the same. There's no need for those backslashes.
Just look at any Groovy file in the repository, e.g.: https://github.com/Netflix/genie/blob/master/genie-agent/src/test/groovy/com/netflix/genie/agent/execution/statemachine/StateMachineConfigSpec.groovy#L43-L53

Reply:
I am not saying that this is a string. Hence removed the `\`. I was trying to tell you how I ended up adding those. Groovy and java do behave differently if you try to write a multi line string(Example -    
 String data = "a" + "b" 
    + "c"
). The link you provided is for a method call. There `\` will not be needed.

Reply:
As I mentioned earlier, I was just trying to give the context and trying to make it clear that Java and groovy do behave differently. The slashes are already removed.